util/stmtsummary/statement_summary.go

```diff
@@ -72,6 +72,19 @@ type stmtSummaryByDigestMap struct {
 
 	// sysVars encapsulates system variables needed to control statement summary.
 	sysVars *systemVars
+
+	evictedInfo stmtSummaryEvictedInfo
```

Comment:
how about putting this into `evictedInfoHistory` and rename to `currentInfo` ?

Reply:
Why not😄